src/lib/connection.js

```diff
@@ -270,6 +271,13 @@ export default class Connection extends EventEmitter {
     this.removeAllListeners()
   }
 
+  sendPing() {
```

Comment:
I would rename this to `pingAndReschedule` or something, so that it is obvious that calling this starts the periodic pinging and doesn't just ping once.